python/Types.sip

```diff
@@ -94,7 +94,7 @@
         for (int i = 0; i < (int)sipCpp->size(); ++i)
         {
             SceneNode *cpp = new SceneNode(*sipCpp->at(i));
-            PyObject *pobj = sipConvertFromInstance(cpp, sipClass_SceneNode, sipTransferObj);
+            PyObject *pobj = sipConvertFromType(cpp, sipType_SceneNode, sipTransferObj);
```

Comment:
@Ghostkeeper 
This is just an update from Sip ABI 11 (Sip <= 4.18.x) to Sip ABI 12 (Sip >= 4.19.0). As Sip 4.19.2 is the minimum required version, this should be correct anyway (and removes deprecated code).

Reply:
Its even much older, sipConvertFromInstance is deprecated since 4.8.0!